charts/leonardo/values.yaml

```diff
@@ -1,6 +1,9 @@
 global:
   # global.applicationVersion -- What version of the Leonardo application to deploy
   applicationVersion: latest
+# Use a k8s instance of mysql for database, used for preview environments.
```

Comment:
Can you move this to be under deployments?

Reply:
I was about to do this and then realized that doesn't work for this variable. the localMysql actually sets up a totally different set of containers statefulset/service/etc and doesn't actually change the deployment at all. (The config changes to the deployment actually happen in firecloud-develop configs.)